test/core_tests.js

```diff
@@ -0,0 +1,138 @@
+QUnit.begin(function( details ) {
+  // In case tests didn't complete, we are left with a hash/anchor pointing somewhere. But we want to start from scratch:
+  window.location.hash = "";    
+});
+
+QUnit.test( "Initialize Impress.js", function( assert ) {
+  assert.ok( impress, 
+             "impress declared in global scope" );
+  assert.strictEqual( impress().init(), undefined,
+                    "impress().init() called." );
+  assert.strictEqual( impress().init(), undefined,
```

Comment:
If my assumption above is correct, then we don't need to assert the API return, which is void and therefore shouldn't be relied upon. Testing a feature marks it as supported, which can cause confusion. We should have green light to change the `impress.init()` return in the future, if necessary.